src/data-dictionary/IgSummary.ts

```diff
@@ -252,7 +253,12 @@ export class IgSummary {
     public async generateSpreadsheet() {
         // Get external dependencies
         await this.getExternalDefs();
-
+        let excludeElements = [
```

Comment:
are these exclusion elements going to be specific to ig's run through this tool? If so, will need to turn this into a configuration option or add inference logic long-term.

Reply:
There's already support for a configuration file. This list could be taken from there and you have a quick and dirty way to configure by IG. Add a few instructions on the types of errors that would suggest you might want to do this and you're done for a first pass. I think that would be valuable and needed for a goal of allowing this to be run on arbitrary MADFSH IGs.

Reply:
Change it to make in configuration and updated readme